### PR TITLE
Add RQM Course Coordinator

### PR DESCRIPTION
**Pull request by linamnt**

### CODE REVIEW

File: Council_Roles.md [hunk 1]
```diff
@@ -2,53 +2,44 @@
...
 ### Special events coordinator
 
 1. Responsible for coordinating and organizing:
     - Software Carpentry workshops throughout the year (at least one per term)
     - Workshops with various organizations (e.g. CUPE)
     - Any other coding type events that come up (e.g. hackathons, code sprints)
+    
+### RQM Course Coordinator
+
+1. Responsible for coordinating and organizing the Reproducible Quantitative Methods (RQM) course (based on [the course by Christie Bahlai](https://cbahlai.github.io/rqm-template/)) at the University of Toronto with the help of the other UofT Coders executive council members,
```
Comment (lwjohnst86) [inline Council_Roles.md]: Should be a period at the end of the sentence

Comment (linamnt) [inline Council_Roles.md]: follwing the others, should have a semicolon, is that okay?

File: Council_Roles.md [hunk 2]
```diff
@@ -2,53 +2,44 @@
...
+3. Ensure that UofT Coders affairs are conducted in a fiscally responsible manner;
+4. Marketing:
   * Send out recruitment emails to various avenues to increase awareness
 5. Make final decisions in the event where decisions are split;
 
 ### Treasurer
 
-1.  Act as UofT Coders financial officer, and together with the
-President, ensure the groups financial stability;
+1.  Act as UofT Coders financial officer, and together with the President, ensure the groups financial stability;
```
Comment (QuLogic) [inline Council_Roles.md]: group's

File: Council_Roles.md [hunk 3]
```diff
@@ -2,53 +2,44 @@
...
-policies established by Student Affairs, University of
-Toronto;
+3. Keep receipts for all financial transactions, and record on the [council repository](https://github.com/UofTCoders/council/tree/master/treasurer) all debits, credits, and cheques released, as well as the dates and remaining bank balance for maintaining and organizing bank records, and for conducting annual audits (if applicable) in accordance with policies established by Student Affairs, University of Toronto;
 4. Apply for funding and sponsorship for the group, as applicable;
 5. Prepare an annual budget at the start of fiscal year;
-6. Present current statements of accounts versus budget,
-and cash flow positions at Executive Council meetings;
-7. Prepare of an annual financial report at the end of the
-fiscal year.
+6. Present current statements of accounts versus budget, and cash flow positions at Executive Council meetings;
```
Comment (QuLogic) [inline Council_Roles.md]: Hmm, have we ever actually done that at a meeting?

File: Council_Roles.md [hunk 4]
```diff
@@ -2,53 +2,44 @@
...
+7. Prepare of an annual financial report at the end of the fiscal year.
 
 ### Technical Admin
 
 1. Responsible for the maintenance and upkeep of the [website](uoftcoders.github.io/studyGroup/).
```
Comment (QuLogic) [inline Council_Roles.md]: https:// missing (not sure if it links properly as is) + semicolon

File: Council_Roles.md [hunk 5]
```diff
@@ -2,53 +2,44 @@
...
 ### Special events coordinator
 
 1. Responsible for coordinating and organizing:
     - Software Carpentry workshops throughout the year (at least one per term)
     - Workshops with various organizations (e.g. CUPE)
     - Any other coding type events that come up (e.g. hackathons, code sprints)
+    
+### RQM Course Coordinator
+
+1. Responsible for coordinating and organizing the Reproducible Quantitative Methods (RQM) course (based on [the course by Christie Bahlai](https://cbahlai.github.io/rqm-template/)) at the University of Toronto with the help of the other UofT Coders executive council members;
```
Comment (QuLogic) [inline Council_Roles.md]: Copy link to other PR?

File: Council_Roles.md (hunk 3 above)
Comment (lwjohnst86) [inline Council_Roles.md]: Hmm, good point... given that there is a README always available, number 7. doesn't make sense... However, number 6. is really important. I think we should do this soon, maybe at the next meeting (not tomorrow).

Comment (linamnt) [inline Council_Roles.md]: Okay I will keep it and maybe take out 7?

Comment (mbonsma) [inline Council_Roles.md]: That's fine with me (take out 7 and keep 6).

**Pull request merged**